quipucords/api/messages.py

```diff
@@ -195,3 +195,6 @@
     'the invalid filter key %s.'
 REPORT_INVALID_FILTER_QUERY_PARAM = 'One or more filter keys are not '\
     'attributes of the system fingerprint: %s.'
+REPORTS_MISSING_REQUIRED_VALUE = 'A required value for Reports was missing.'
```

Comment:
I feel like this error message is vague. This is printed if a `report_id` and `details.json` and `deployments.json` are all `None`. It seems like the report does not exist at that point but I'm not sure?